robottelo/config/validators.py

```diff
@@ -279,6 +279,22 @@
             must_exist=True,
         )
     ],
+    remotedb=[
+        Validator(
+            'remotedb.server',
+            'remotedb.db_server',
+            'remotedb.common_db_password',
+            must_exist=False,
```

Comment:
```suggestion
            must_exist=True,
```
??

Reply:
keeping this false, as this fields will required only while testing remote db's not else